scripts/seed.py

```diff
@@ -0,0 +1,210 @@
+# Command to run : python manage.py shell --settings=settings.dev  < scripts/seed.py
+
+from django.conf import settings
+from django.core.files.uploadedfile import SimpleUploadedFile
+from django.contrib.auth.models import User
+from django.utils import timezone
+
+from allauth.account.models import EmailAddress
+
+from datetime import timedelta
+from faker import Factory
+
+import os
+
+from challenges.models import Challenge, ChallengePhase, DatasetSplit, Leaderboard, ChallengePhaseSplit
+from hosts.models import ChallengeHostTeam, ChallengeHost
+from participants.models import Participant, ParticipantTeam
+
+
+def create_admin_user():
+    username = "admin"
+    password = "password"
+    email = "admin@gmail.com"
+    user = User.objects.create_user(
+        email=email,
+        username=username,
+        password=password,
+        is_staff=True,
+        is_superuser=True,
+    )
+    EmailAddress.objects.create(user=user, email=email, verified=True, primary=True)
+    print "Super user created with \n username: %s \n password: %s" % (username, password)
+
+
+def create_user(username=""):
+    Factory.create()
+    password = "password"
+    email = "%s@gmail.com" % (username)
+    user = User.objects.create_user(
+        email=email,
+        username=username,
+        password=password,
+    )
+    EmailAddress.objects.create(user=user, email=email, verified=True, primary=True)
+    print "User created with \n username: %s \n email: %s \n password: %s" % (username, email, password)
+    return user
+
+
+def create_challenge_host_team(user):
+    fake = Factory.create()
```

Comment:
We will be creating Factory object in each function. Can you actually check and see if it no harm creating a single global object for Factory and then using it everywhere in the function?

As per my understanding it should work, but you can update me if it does not.